llm_studio/src/datasets/text_causal_language_modeling_ds.py

```diff
@@ -31,6 +31,9 @@ def __init__(self, df: pd.DataFrame, cfg: Any, mode: str = "train"):
         self.tokenizer = get_tokenizer(self.cfg)
         self.conversation_chain_handler = ConversationChainHandler(self.df, cfg)
 
+        if cfg.type == "causal_classification":
```

Comment:
We could also think about subclassing the dataset. As the code change is minimal atm, I'm also fine with leaving it at is.